src/index.ts

```diff
@@ -18,4 +18,5 @@ export function provideAnalytics(config: AnalyticsConfig): AnalyticsReporter {
   return new AnalyticsReporter(config, httpRequester);
 }
 
-export * from './models';
\ No newline at end of file
+export * from './models';
+export { AnalyticsReporter };
```

Comment:
do we need to export AnalyticsReporter if user can invoke provideAnalytics?

Reply:
yes, we need it to add typing when defining context for it and for what's returned by a `useAnalytics` hook (see use in the component library [PR](https://github.com/yext/answers-react-components/pull/17))

Reply:
Does exporting AnalyticsService work instead?

Reply:
it might even make sense for provideAnalytics to have return type `AnalyticsService` instead

Reply:
if I'm being perfectly honest I'm not sure what the `AnalyticsService` interface existence gets us in the first place besides letting us rename `AnalyticsReporter` and add arbitrary non-private properties, but this is probably just fussing over minutiae

Reply:
yeah, `AnalyticsService` works. I can use that instead for `provideAnalytics`